flink-table/flink-sql-gateway/src/main/java/org/apache/flink/table/gateway/rest/SqlGatewayRestEndpoint.java

```diff
@@ -37,6 +46,7 @@
 public class SqlGatewayRestEndpoint extends RestServerEndpoint implements SqlGatewayEndpoint {
 
     public final SqlGatewayService service;
+    protected Time timeout = Time.seconds(1);
```

Comment:
This should be configurable.

Reply:
And why is this `protected` and not `final`?

Reply:
This attribute is not required currently.